### ISSUE TITLE
tab-index is missing, accessibility issue

### ISSUE THREAD

**Issue by omouse** [react-component/time-picker#33]
When I press tab and the time-picker is open, it tabs over to another component instead of tabbing through and letting me use the arrow keys to select a time. I also can't seem to type in any numbers?

**Comment by afc163** [comment]
PR is welcome~

**Comment by omouse** [comment]
no longer working with react, closing my issue, thx!
